cordial_face/scripts/setup_http_servers.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+killall http-server
+
+source /opt/ros/kinetic/setup.bash
+source /home/qtrobot/catkin_ws/devel/setup.bash
```

Comment:
this might be able to be made more general with some kind of environment variables